app/parse.py

```diff
@@ -8,8 +18,66 @@ class Quote:
     tags: list[str]
 
 
+QUOTE_FIELDS = [field.name for field in fields(Quote)]
+
+
+logging.basicConfig(
+    level=logging.INFO,
+    format="[%(asctime)s] - [%(levelname)8s]: %(message)s",
+    handlers=[
+        logging.StreamHandler(sys.stdout),
+        logging.FileHandler("parser.log"),
+    ]
+)
+
+
+def parse_single_quote(quote_soup: BeautifulSoup) -> Quote:
+    text = quote_soup.select_one(".text").text
+    author = quote_soup.select_one(".author").text
+    tags = (
+        quote_soup.select_one(".tags").text
+        .replace(" ", "").split("\n")
+    )
+    tags = [tag for tag in tags if tag and tag != "Tags:"]
```

Comment:
The logic for extracting tags assumes that tags are separated by spaces and newlines. If the HTML structure changes, this might break. Consider using a more robust method to extract tags, such as selecting all tag elements and iterating over them.